WooCommerce/Classes/POS/Presentation/Item Selector/ItemList.swift

```diff
@@ -79,8 +79,9 @@ private enum Constants {
 @available(iOS 17.0, *)
 private struct ItemListRow: View {
     let item: POSItem
-    let analytics: Analytics = ServiceLocator.analytics
+    let node: ItemListBaseItem
```

Comment:
As I see `node` is only needed here to choose a correct stack. If we remove `node.itemType` check, we may be able to remove `node` from here.